mini-bots/LuaManagment/DiscordLua.cs

```diff
@@ -21,9 +21,11 @@ public string Run(int id, string code, DiscordMessage message, DatabaseManager d
             _lua["messageManager"] = new MessageManager(message);
             _lua["timeManager"] = new TimeManager();
             _lua["storageManager"] = new StorageManager(databaseManager, id);
+            _lua["jsonManager"] = new JsonManager(this);
 
             Byte[] luaIn = Encoding.UTF8.GetBytes(code);
 
+            _lua.State.Encoding = Encoding.UTF8;
```

Comment:
not really related, but we should check this out if it can produce utf8 output

Reply:
removed it